src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -276,6 +277,18 @@ function MoneyRequestView({
         [transactionAmount, isSettled, isCancelled, isPolicyExpenseChat, isEmptyMerchant, transactionDate, hasErrors, hasViolations, translate, getViolationsForField],
     );
 
+    const deleteTransaction = useCallback(() => {
+        if (!parentReportAction) {
+            return;
+        }
+        const iouTransactionID = parentReportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU ? parentReportAction.originalMessage?.IOUTransactionID ?? '' : '';
+        if (ReportActionsUtils.isTrackExpenseAction(parentReportAction)) {
+            IOU.deleteTrackExpense(parentReport?.reportID ?? '', iouTransactionID, parentReportAction, true);
```

Comment:
See [Default value for inexistent IDs](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#default-value-for-inexistent-IDs)
```suggestion
            IOU.deleteTrackExpense(parentReport?.reportID ?? '-1', iouTransactionID, parentReportAction, true);
```